OWNERS_ALIASES

```diff
@@ -7,21 +7,23 @@ aliases:
     - neolit123
     - timothysc
   cluster-api-admins:
-    - davidewatson
-    - detiber
     - justinsb
+    - detiber
     - vincepri
```

Comment:
```suggestion
    - detiber
    - justinsb
    - vincepri
```

Reply:
ah I was lazy and copy pasted from CAPI's file... will fix